web-server/v0.4/src/models/dashboard.js

```diff
@@ -167,25 +175,25 @@ export default {
     *updateSelectedController({ payload }, { select, put }) {
       yield put({
         type: 'modifySelectedController',
-        payload: payload,
+        payload,
```

Comment:
Is this still valid?  Isn't this supposed to be an object of the `payload` field that is `put()`?

Reply:
This is a feature of ECMAScript 6 that allows you to define object literal methods and properties in a shorthand form for a cleaner syntax. This requires the key name to match the name of the assigned variable.